ristretto/ristretto_test.go

```diff
@@ -0,0 +1,122 @@
+package ristretto_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gofiber/storage/ristretto"
+	"github.com/gofiber/utils"
+)
+
+var testStore = ristretto.New()
+
+func Test_Ristretto_Set(t *testing.T) {
+	var (
+		key = "john"
+		val = []byte("doe")
+	)
+
+	err := testStore.Set(key, val, 0)
+	utils.AssertEqual(t, nil, err)
+}
+
+func Test_Ristretto_Set_Override(t *testing.T) {
+	var (
+		key = "john"
+		val = []byte("doe")
+	)
+
+	err := testStore.Set(key, val, 0)
+	utils.AssertEqual(t, nil, err)
+
+	err = testStore.Set(key, val, 0)
+	utils.AssertEqual(t, nil, err)
+}
+
+func Test_Ristretto_Get(t *testing.T) {
+	var (
+		key = "john"
+		val = []byte("doe")
+	)
+
+	err := testStore.Set(key, val, 0)
+	utils.AssertEqual(t, nil, err)
+
+	result, err := testStore.Get(key)
+	utils.AssertEqual(t, nil, err)
+	utils.AssertEqual(t, val, result)
+}
+
+func Test_Ristretto_Set_Expiration(t *testing.T) {
+	var (
+		key = "john"
+		val = []byte("doe")
+		exp = 1 * time.Second
+	)
+
+	err := testStore.Set(key, val, exp)
+	utils.AssertEqual(t, nil, err)
+
+	time.Sleep(1100 * time.Millisecond)
```

Comment:
I assume this line suspends the rest of the tests until the value is expired? What about using `testStore.Reset()` instead?

Reply:
Yeah your assumption is correct, we can just simply remove this one as `testStore.Reset()` will do the same.